### PR TITLE
25 vote set threshold

### PR DESCRIPTION
**Pull request by dwalintukan**
#25

### CODE REVIEW

File: contracts/event/EventFactory.sol [hunk 1]
```diff
@@ -16,7 +16,7 @@ contract EventFactory is NRC223Receiver {
         uint amount;
     }
 
-    uint16 private constant VERSION = 5;
+    uint16 private constant VERSION = 6;
```
Comment (TT1943) [inline contracts/event/EventFactory.sol]: can we create a branch called `version-6` and branch off all these changes into that branch, and merge deploy all together when everything is done.

Comment (TT1943) [inline contracts/event/EventFactory.sol]: can we create a branch called `version-6` and branch off all these changes into that branch, and merge deploy all together when everything is done.

Comment (dwalintukan) [reply contracts/event/EventFactory.sol]: what other changes need to go into this version?

Comment (dwalintukan) [reply contracts/event/EventFactory.sol]: done

**Pull request merged**